TLM/TLM/State/OptionsTabs/PoliciesTab_InEmergenciesGroup.cs

```diff
@@ -0,0 +1,32 @@
+namespace TrafficManager.State {
+    using ICities;
+    using TrafficManager.UI;
+    using TrafficManager.UI.Helpers;
+
+    public static class PoliciesTab_InEmergenciesGroup {
+
+        public static CheckboxOption EvacBussesMayIgnoreRules =
+            new (nameof(Options.evacBussesMayIgnoreRules), Options.PersistTo.Savegame) {
+                Label = "VR.Checkbox:Evacuation buses may ignore traffic rules",
+                Validator = NaturalDisastersDlcValidator,
+            };
+
+        private static bool HasNaturalDisastersDLC
+            => SteamHelper.IsDLCOwned(SteamHelper.DLC.NaturalDisastersDLC);
+
+        internal static void AddUI(UIHelperBase tab) {
+            if (!HasNaturalDisastersDLC) return;
```

Comment:
Would it be a good idea to show the group caption at all times, but then if DLC is not owned, add a text label explaining that a DLC is required?

Reply:
Yes, I'm thinking of doing that in later iteration once I add some more UI components.